tests/sources/test_servicenow.py

```diff
@@ -769,3 +797,116 @@ async def test_get_docs_with_advanced_rules(filtering):
                 "_timestamp": "1212-12-12T12:12:12",
             },
         ] == response_list
+
+
+@pytest.mark.asyncio
+async def test_get_access_control():
+    async with create_service_now_source() as source:
+        source.servicenow_client.get_table_length = mock.AsyncMock(return_value=2)
+        source._dls_enabled = Mock(return_value=True)
+        with mock.patch.object(
+            ServiceNowClient,
+            "get_data",
+            return_value=AsyncIterator(
+                [
+                    [
+                        {
+                            "sys_updated_on": "2023-10-10 05:21:45",
+                            "sys_id": "id_1",
+                            "email": "admin@email.com",
+                            "user_name": "demo.user",
+                        }
+                    ]
+                ]
+            ),
+        ):
+            async for user in source.get_access_control():
+                assert user["_id"] == "id_1"
```

Comment:
Can you make this a more robust assertion about the contents of `user`?

Reply:
yes, checked full response